manifests/install.pp

```diff
@@ -25,6 +27,16 @@
 
   case $ensure {
     'present', 'installed', default: {
+      if $require_verify {
+        exec { "yum-reinstall-${name}":
+          command => "yum -y reinstall '${source}'",
+          onlyif  => "rpm -q '${name}'",
+          unless  => "rpm -V '${name}'",
+          timeout => $timeout,
+          before  => Exec["yum-install-${name}"],
```

Comment:
does that make sense? Do we want to run `yum install` after we did `zum reinsstall`?

Reply:
In theory, the `yum install` should detect the rpm is loaded (via the `rpm -q` check (https://github.com/voxpupuli/puppet-yum/blob/master/manifests/install.pp#L30) and just skip it.